lib/rack/utils.rb

```diff
@@ -551,6 +551,7 @@ def names
       428 => 'Precondition Required',
       429 => 'Too Many Requests',
       431 => 'Request Header Fields Too Large',
+      451 => 'Unavailable For Legal Reasons',
```

Comment:
Would be appropriate to mention the exception at https://github.com/rack/rack/pull/985#discussion-diff-48199164R501 since the comment says exactly how this list is generated.